test/Npgsql.Tests/ReaderNewSchemaTests.cs

```diff
@@ -565,6 +565,89 @@ public async Task PostgresType()
             }
         }
 
+        [Test]
+        public async Task ColumnSchemaWithoutWithNoKeyInfo()
+        {
+            await using var conn = await OpenConnectionAsync();
+            await using var _ = await CreateTempTable(conn, "foo INTEGER", out var table);
+
+            using var cmd = new NpgsqlCommand($"SELECT foo, foo AS foobar, 8 AS bar, 8, '8'::VARCHAR(10) FROM {table}", conn);
+            await using var reader = await cmd.ExecuteReaderAsync();
+            var columns = reader.GetColumnSchema();
+
+            Assert.That(columns[0].ColumnName, Is.EqualTo("foo"));
+            Assert.That(columns[0].BaseColumnName, Is.EqualTo(null));
```

Comment:
FYI, there is `Is.Null` (:

Reply:
Thanks @YohDeadfall, got a little tunnel vision with all the copy and paste.